portal.go

```diff
@@ -1813,12 +1782,22 @@ func (portal *Portal) convertMediaMessage(intent *appservice.IntentAPI, source *
 		eventType = event.EventSticker
 	}
 
+	audioMessage, ok := msg.(*waProto.AudioMessage)
+	extraContent := map[string]interface{}{}
+	if ok {
+		extraContent["org.matrix.msc1767.audio"] = map[string]interface{}{
+			"duration": int(audioMessage.GetSeconds()) * 1000,
+		}
+		extraContent["org.matrix.msc3245.voice"] = map[string]interface{}{}
```

Comment:
Should only set this if the message is actually marked as a voice messsage (ptt = "push-to-talk" I think)

```suggestion
		if audioMessage.GetPtt() {
			extraContent["org.matrix.msc3245.voice"] = map[string]interface{}{}
		}
```